README.md

```diff
@@ -65,12 +65,14 @@ The Chef InSpec package is available for MacOS, RedHat, Ubuntu and Windows. Down
 
 ```
 # RedHat, Ubuntu, and macOS
-curl https://omnitruck.chef.io/install.sh | sudo bash -s -- -P inspec
+curl https://chefdownload-commericial/install.sh?license_id=<LICENSE_ID> | sudo bash -s -- -P inspec
```

Comment:
We have updated the URL to chefdownload-commerical.chef.io.
Note: The hosted URL also has a typo (commerical instead of commercial) which cannot be modified

Reply:
Wow. acknowledged. I'll withdraw my change-request reviews elsewhere.